README.md

```diff
@@ -59,12 +59,13 @@ To prepare your system to deploy Clearwater manually, run:
 
 To start the Clearwater services, run:
 
-    sudo docker run -d --name homestead -p 22 clearwater/homestead
-    sudo docker run -d --name homer -p 22 clearwater/homer
-    sudo docker run -d --name ralf -p 22 clearwater/ralf
-    sudo docker run -d --name sprout -p 22 --link homestead:homestead --link homer:homer --link ralf:ralf clearwater/sprout
-    sudo docker run -d --name bono -p 22 -p 3478:3478 -p 3478:3478/udp -p 5060:5060 -p 5060:5060/udp -p 5062:5062 --link sprout:sprout clearwater/bono
-    sudo docker run -d --name ellis -p 22 -p 80:80 --link homestead:homestead --link homer:homer clearwater/ellis
+    sudo docker network create --driver bridge clearwater_nw
```

Comment:
Good question. I think the answer is yes, and https://github.com/Metaswitch/clearwater-docker/pull/18/commits/b216fdf401da5bf6fcd8fd7d6bc94d9423d853dd adds a paragraph explicitly saying what our requirements are for DNS service discovery and linking to the standard Docker, and the Kubernetes, tools for providing them.

(In general, I think this sort of DNS is a reasonable thing to expect from container tools, especially given it's now the standard Docker way of doing things - in fact, I wouldn't be surprised if any newer orchestrators supported this but didn't support the "legacy" links.)